com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/UpdateAppStepTest.java

```diff
@@ -199,6 +245,24 @@ private void prepareDiscontinuedServices() {
         notRequiredServices.addAll(discontinuedServices);
     }
 
+    private boolean listContainsElement(List<String> list, String element) {

```

Comment:
Why not using List::contains method :D ?

Reply:
Done